### PR TITLE
Admin Staff - Bugs, Fixes & Enhancements (#131 & #132)

### PR DESCRIPTION
**Pull request by varshasivakumar**
**Bugs & Fixes:**

1. Padding after Accept/Reject button
2. Remove extra “>” from detailed view
3. The dashboard tab header should be similar to student dashboard tab header
4. ‘School Informations’ tab header should be corrected to ‘School Information’
5. View application - status label should have ‘s’ capital.
6. Use broken html page template for error pages while using invalid URL

**Enhancements:**

1. Make tabs clickable
2. Implement a back button tab from admin dashboard detail page to index page
3. Re-implement Program filter feature
4. Map M/F to Male Female for detailed view
5. Re-Structure JS files under static files

### CODE REVIEW

File: dashboard/templates/dashboard/index.html [hunk 1]
```diff
@@ -5,8 +5,10 @@
     <!-- Load bootstrap css, jquery, bootstrap js -->
     <!-- common across all applications inheriting from dashboard -->
     <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>
+    <script src="{% static 'js/jquery-3.1.1.slim.min.js' %}"></script>
+    <script src="{% static 'js/popper.js' %}"></script>
+{#    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>#}
```
Comment (abe-shek) [inline dashboard/templates/dashboard/index.html]: Since the functionality is tested with a local min.js file, we can have this comment removed.

File: dashboard/templates/dashboard/index.html [hunk 2]
```diff
@@ -65,12 +67,21 @@
...
-            <nav class="nav d-flex">
-                <a class="p-2 text-muted active" href="{% url 'dashboard:admissions:index' %}"
-                   data-toggle="tab">Student
-                    Applications</a>
-                <a class="p-2 text-muted" href="#school_informations" data-toggle="tab">School
-                    Informations</a>
+            <nav class="nav d-flex justify-content-between">
+                <div class="nav nav-tabs d-flex justify-content-between" id="dashboard-tab"
+                     role="tablist" style="width: 100%;">
+                    {# TODO: update this    #}
```
Comment (abe-shek) [inline dashboard/templates/dashboard/index.html]: I'm a bit confused about this todo, it'll be helpful if you can elaborate it a bit more.

File: dashboard/templates/dashboard/index.html (hunk 1 above)
Comment (varshasivakumar) [reply dashboard/templates/dashboard/index.html]: I just kept that for reference. I could remove the comment.

File: dashboard/templates/dashboard/index.html (hunk 2 above)
Comment (varshasivakumar) [reply dashboard/templates/dashboard/index.html]: This Todo was already present for student tab. Since I was supposed to replicate the same for admin staff, I let the comment be the same too. Not sure who added it for student or what it means.

**Comment by varshasivakumar** [comment]
> Overall, the functionality that is expected for the Admissions staff and viewing applications is there but a few things that need clarification:
> 
> 1. Like Abe said, the student application no longer allows you to choose a program.
> 2. The PR mentions changing "‘School Informations’ tab header should be corrected to ‘School Information’" but I noticed that tab was actually removed.

The School Information tab as of now shows no information and wouldn't take us anywhere if we clicked on it. So for this sprint we'll remove tabs that haven't been implemented yet (More Info on all schools tab & School information tab in admin dashboard) and bring them back next sprint with it's full functionality.

**Comment by varshasivakumar** [comment]
> * There seem to be some styling issues with the unauth page - "no-school-placeholder" class never gets its CSS values in the dom resulting in the issue.
> * Additionally, the "application/detail" page kinda escapes from the unauth check (when trying to access "http://127.0.0.1:8000/dashboard/ut_admin_staff/detail/7" as a student user).
> 
> The rest is covered and the empty list page looks good and provides for consistency.

Fixed.

**Comment by varshasivakumar** [comment]
> LGTM, but a few points need a bit more clarification -
> 
> * the application form page for student is behaving erratically with this PR. I'm not able to select a school properly and furthermore the programs for the selected school are never updated. Not sure how it might've been impacted by this PR since there's not much that has changed, except for the jquery and js files.
> * stylizing the empty results pages from manipulating the URL for the detail view and the program filter would add to the consistency across the website (similar to all schools tab)
> * lastly and this might be a nit, the background color for the body in admissions application is set to the default value from bootstrap.min.css (#fff), while on student side it's set to #f5f5f5. So if we're not changing the theme for admin side and sticking to the same as that of the student, maybe we can update this. This will also help in getting rid of the extra gray box that appears on the active tab link...making it consistent across the website.
> 
> I do have some screenshots if something needs more clarity, so please do let me know.

Fixed.
